app/src/test/java/org/mozilla/focus/utils/UrlUtilsTest.java

```diff
@@ -126,6 +142,11 @@ public void testStripUserInfo() {
         assertEquals("ftp://mozilla.org", UrlUtils.stripUserInfo("ftp://user:password@mozilla.org"));
 
         assertEquals("öäü102ß", UrlUtils.stripUserInfo("öäü102ß"));
+
+        String badUri = "https://user:password@www.i/have/percentage/signs/%.org%";
```

Comment:
This fails because it's not encoded correctly right (e.g. % should be associated with a numeric encoding). Can you change the uri to make that clearer? "I have percent signs" wasn't clear.